ra2ce/analysis/losses/losses_base.py

```diff
@@ -138,12 +138,14 @@ def _get_disrupted_criticality_analysis_results(
             criticality_analysis = criticality_analysis.drop_duplicates(["u", "v"])
         #  ToDO: check hazard overlay with AggregateWlEnum.NONE or INVALID
 
-        aggregate_wl_suffix = self.analysis_config.config_data.aggregate_wl.get_suffix()
+        aggregate_wl_prefix = (
+            self.analysis_config.config_data.aggregate_wl.get_wl_prefix()
+        )
         hazard_aggregate_wl_columns = [
             c
             for c in criticality_analysis.columns
             if (c.startswith("RP") or c.startswith("EV"))
-            and c.endswith(f"_{aggregate_wl_suffix}")
+            and c.endswith(f"_{aggregate_wl_prefix}")
```

Comment:
Why call it a prefix if it seems to be a suffix (`endswith`)?

Reply:
It is the suffix of aggregate_wl Enum. example: MAX => ma